src/components/FormView.jsx

```diff
@@ -14,6 +13,11 @@ import { getFieldName } from './utils';
 import Field from 'volto-form-block/components/Field';
 import './FormView.css';
 
+const HCaptcha = loadable(() => import('@hcaptcha/react-hcaptcha'));
+const { GoogleReCaptcha } = loadable.lib(() =>
+  import('react-google-recaptcha-v3'),
```

Comment:
Here you should provide a fallback. (I think).

And I hope I'm not mistaking, but this is one of the reasons that I've created the injectLazyLibs in Volto, otherwise you have to check if GoogleReCaptcha has loaded, lower in the code. See https://docs.voltocms.com/recipes/lazyload/#lazy-loading-libraries

Reply:
I refactored this a lot and I'm now using injectLazyLibs, in a  component conditionally rendered so I'm not calling that loadable library if not needed.